### DISCUSSION TITLE
main code

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1423] [Q&A]
Hi, I want to know which the code is the crazyflie main code to run whole system ? tks

**Comment by gemenerik** [comment]
I am not sure what you mean. As you can see we have several repositories. There is no one repository that includes all the code. I think reading the repository descriptions will give you some idea on what they include, otherwise you can read the documentation. Hope that helps you get started.

EDIT: perhaps you mean the firmware that runs on the STM32? That's `crazyflie-firmware`.
